lib/plugins/aws/package/compile/functions/index.test.js

```diff
@@ -2216,6 +2216,29 @@ describe('AwsCompileFunctions', () => {
         ).to.not.exist;
       });
     });
+
+    it('should only add an SLS_LAMBDA_VERSION_DIGEST env var to versioned functions', () => {
```

Comment:
Ideally all new tests should be based on`runServerless` util. See https://github.com/serverless/serverless/tree/master/test#unit-tests

And in this file at the bottom we already have some tests that rely on that, let's add over there tests that confirms on that env var.